sentry_streams/sentry_streams/examples/alerts.py

```diff
@@ -25,19 +30,17 @@
     function=build_event,
 )
 
-# Windows are set to be open for 5 seconds
-reduce_window = SlidingWindow(window_size=timedelta(seconds=5), window_slide=timedelta(seconds=2))
+flat_map = FlatMap(name="myflatmap", ctx=pipeline, inputs=[map], function=materialize_alerts)
```

Comment:
Please add a comment that explains how alerts would work. It is not immediately obvious why there is a flatMap here.